gravitee-apim-e2e/api-test/src/gateway/redeploy-api.spec.ts

```diff
@@ -108,8 +108,6 @@ describe('Redeploy Api', () => {
       // Execute a request during which we will redeploy the api (only one try, no delay before call)
       const fetchUniqueGatewayCall = fetchGatewaySuccess({
         contextPath: createdApi.context_path + '?activeLatency=true',
-        failAfterMs: 0,
```

Comment:
maxRetries. because here we dont't want retry

Reply:
no ? 🤔

Reply:
0 does not work with this mechanism, but one does I'll wait for the current build to see how it goes and update after that